RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
  0%|          | 0/100 [00:00<?, ?it/s]  1%|          | 1/100 [00:47<1:17:48, 47.16s/it]  4%|▍         | 4/100 [01:06<22:33, 14.09s/it]   65%|██████▌   | 65/100 [01:10<00:22,  1.57it/s] 68%|██████▊   | 68/100 [01:10<00:19,  1.66it/s] 73%|███████▎  | 73/100 [01:16<00:18,  1.49it/s] 76%|███████▌  | 76/100 [01:16<00:14,  1.60it/s]100%|██████████| 100/100 [01:16<00:00,  1.30it/s]
Exception ignored in: <function Pool.__del__ at 0x2b3b24824160>
Traceback (most recent call last):
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/multiprocessing/pool.py", line 268, in __del__
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/multiprocessing/queues.py", line 362, in put
AttributeError: 'NoneType' object has no attribute 'dumps'
